### ISSUE TITLE
As a scholar, I can create a private collection and add collection metadata

### ISSUE THREAD

**Issue by roelandordelman** [CLARIAH/wp5_mediasuite#290]

**Comment by jblom** [comment]
@jwassenaarbg @roelandordelman I put this in review/QA, because I need to test what Johannes made for this. Coming back to this later

**Comment by jblom** [comment]
@jwassenaarbg ok I've integrated your stuff into the master of the mediasuite & components repos. Except for obvious styling/UX stuff, it seems to work fine. 

Now only non Open Conext users see the personal collection pages. After we've fixed the UX and taken a few extra steps (t.b.d.) we can offer it to CLARIAH users as well.

Also ping @roelandordelman

**Comment by jblom** [comment]
Alright, now I've went through the code a bit more and did a few minor revisions. The nice thing is: it's now possible to play search through your personal collection in the media suite; also enabling you to play & annotate the content.

For this I also had to update the search API for allowing access to collections that you have personally created (Note: now all your personal collections are considered private; even though you've marked it as public).

@roelandordelman @jwassenaarbg I've deployed this stuff in the test environment
